index.Rmd

```diff
@@ -2,7 +2,7 @@
 title: "Opoid-Related Deaths in New York State"
 author: "Anuwat Pengput"
 subtitle: Spatial Epidemiological Analysis and Risk Factors of Opoid-Related Deathsin
-  New York
+  New York 
```

Comment:
Clear narrative to let me know what you want, but there is a typesetting problem, which is that "Deaths" and "in" are not separated by space.